web_tests/pages/login_page.py

```diff
@@ -0,0 +1,69 @@
+from selenium.webdriver.common.by import By
+from web_tests.pages.base_page import BasePage
+
+
+class LogiPage(BasePage):
+
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    USERNAME = (By.XPATH, '//div[@id="app"]//input[@name="username"]')
+    PASSWORD = (By.XPATH, '//div[@id="app"]//input[@name="password"]')
+    LOGIN_BUTTON = (By.XPATH, '//div[@id="app"]//button[@type="submit"]')
+    MAIN_PAGE = (By.XPATH, '//div[@id="app"]//input[@placeholder="Search"]')
+    ERROR_MESSAGE = (By.XPATH, '//div[@id="app"]//p[@class="oxd-text oxd-text--p oxd-alert-content-text"]')
```

Comment:
oxd-text class is used for many elements. Better use more unique with contains check